Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -308,6 +308,10 @@ define([
 
         if (gltfFormat === 0) {
             var gltfUrl = getStringFromTypedArray(gltfView);
+
+            // We need to remove padding from the end of the model URL in case this tile was part of a composite tile
+            // This removes all white space and null characters from the end of the string.
+            gltfUrl = gltfUrl.trim().replace(/\0+$/, '');
```

Comment:
Can't this just be `gltfUrl.replace(/\s+$/, '');`  Why the need for trim or null?